package.json

```diff
@@ -147,5 +147,8 @@
     "website",
     "examples/*",
     "packages/*"
-  ]
+  ],
+  "dependencies": {
+    "reason-react": "^0.7.0"
+  }
```

Comment:
This isn't necessary.
You should add/modify this dep in appropriate package & run `yarn` at the root of the repo. Yarn _workspace_ feature handle all the necessary work to correctly install deps & make symlink if necessary.